ui/src/components/facets/FacetsGrid.js

```diff
@@ -17,6 +17,7 @@ function facetCard(props, facet) {
     return (
       <HistogramFacet
         facet={facet}
+        updateFacets={props.updateFacets}
```

Comment:
This has the same problem as the tooltip -- it's not possible to select values with low numbers. Maybe if we do the transparent background bar, that could help with this.

Reply:
Noted, I'll fix both in my next PR.